docs/modules/ROOT/pages/deploy_related/integration_nova.adoc

```diff
@@ -21,16 +21,288 @@ We would want to integrate with Nova to support the same set of user operations,
 
 == Review of OpenStack Workflow
 
+nova create a vm will through four stage.
+1. check network options is ok
+2. allocate network resource
+3. rollback if fail
+
+In scenario column on below table, tag by above scenario and features.
+
+nova call neutron apis
+[width="100%",cols="1,1,1,1,1,1,1", options="header"]
```

Comment:
Tried to fix the table width but it is still wider than the page limit. Not a big deal. I would suggest to fix it when you have some extra time.

https://github.com/Gzure/alcor/blob/nova-inti/docs/modules/ROOT/pages/deploy_related/integration_nova.adoc